tests/test_evaluation.py

```diff
@@ -28,11 +29,12 @@ def test_pretrained_gpt_model(tmp_path):
         epsilon=1e-5,
         sample_key="input_ids",
         prediction_key="logits",
-        weight_init=WeightInitailizationConfig(mean=0, std=0.02),
+        weight_init=WeightInitializationConfig(mean=0, std=0.02),
+        poe_type=PositionTypes.NOPE
     )
 
     model = GPT2LLM(**pretrained_config)
-    model.save_pretrained(tmp_path)
+    #model.save_pretrained(tmp_path)
```

Comment:
Does this test still work, if we don't save the model anymore?

Btw. this test gets moved/deleted by #34.